core/jobs/src/test/java/com/cognifide/aet/job/common/comparators/layout/LayoutComparatorTest.java

```diff
@@ -0,0 +1,147 @@
+/**
+ * AET
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.cognifide.aet.job.common.comparators.layout;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.cognifide.aet.job.api.comparator.ComparatorProperties;
+import com.cognifide.aet.job.common.comparators.layout.LayoutComparator;
+import com.cognifide.aet.job.common.comparators.layout.utils.ImageComparisonResult;
+import com.cognifide.aet.vs.ArtifactsDAO;
+import java.util.Optional;
+import org.apache.commons.io.IOUtils;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class LayoutComparatorTest {
+
+  @Mock
+  private ComparatorProperties comparatorProperties;
+
+  @Mock
+  private ArtifactsDAO artifactsDAO;
+
+  @Mock
+  private ImageComparisonResult imageComparisonResult;
+
+  private LayoutComparator layoutComparator;
+
+  @Before
+  public void setUp() {
+    //given
+    this.layoutComparator = new LayoutComparator(this.comparatorProperties, this.artifactsDAO);
+  }
+
+  @Test
+  public void Comparator_hasMaskThresholdWithAcceptableDifference_withoutThreshold_expectFalse() {
```

Comment:
`Comparator_` prefix is not needed in test names - please use [Tests naming convention](methodName_whenStateUnderTest_expectBehavior)